QUEST.tsv

```diff
@@ -1293,7 +1294,7 @@ QUEST_20150317_001293	Agony of the Healer Lady
 QUEST_20150317_001294	Completing the Strength of Soul
 QUEST_20150317_001295	Reluctant Souls
 QUEST_20150317_001296	Memorial Service
-QUEST_20150317_001297	Dangerous Throneweaver
+QUEST_20150317_001297	Dangerous Throne Weaver
```

Comment:
"Throneweaver" is correct according to the wiki.  ( Throneweaver is a spider-like boss.  It appears to have a throne on it's back.  Just what I found out while looking it up, and remember playing during the iCBT.  )

I'd leave it like this for now, to allow for easy find/replace mass changes if they need to be made later.